servicecontrol/installation.md

```diff
@@ -54,13 +54,13 @@ Performance Counters are not installed by the [Platform Installer](/platform/ins
 
 ## Using ServiceControl Management to upgrade ServiceControl instances
 
-ServiceControl Management provides a simple means of setting up one or more instances of the ServiceControl service. For production systems it is recommended to limit the number of instances per machine to one.
+ServiceControl Management provides a simple means of setting up one or more instances of the ServiceControl or ServiceControl Monitoring services. For production systems it is recommended to limit the number of instances of each service per machine to one.
 
 WARNING: The ability to add multiple instances is primarily intended to assist development and test environments.
```

Comment:
We should probably change this to make sure we say that this is only for service control standard, not for the monitoring instances. Thoughts?